engines/director/lingo/lingo-code.cpp

```diff
@@ -1653,7 +1658,7 @@ void LC::call(const Symbol &funcSym, int nargs, bool allowRetVal) {
 
 		uint stackSize = g_lingo->_stack.size();
 
-		if (funcSym.u.bltin != LB::b_return && funcSym.u.bltin != LB::b_value) {
+		if (funcSym.u.bltin != LB::b_return && funcSym.u.bltin != LB::b_value && funcSym.u.bltin != LM::m_perform) {
```

Comment:
I do not like this change. Can we avoid it?

Reply:
Well, I don't think so because what that check need to be avoided to prevent throwing of errors in case where m_perform is supposed to return value, ie something like `set myvar = me(mPerform, "ABC")`,

Through, one solution could be to push a dummy value if allowRetVal is true (this change is to be made inside m_perform itself), however as a consequence of this an extra <Void> variable will be left in stack after finishing execution.